scripts/enterGame.js

```diff
@@ -13,35 +13,33 @@ function start() {
 
 // Changes the title on the UI
 function changeTitle() {
-    title = "Rock Paper Scissors";
+    titleElement = "Rock Paper Scissors";
 }
 
 // Removes the start button from the UI
 function removeUI() {
-    start = "";
+    startElement = "";
 }
 
 // Draws the entire game, by writing in all the new codes.
-function drawUI {
+function drawUI() {
```

Comment:
When I open it, there's this line here `const displayElement = document.getElementById("display");` which doesn't affect the codes at all. If I change it to "drawElement", it would delete the rock paper scissors buttons.

This code doesn't exists in your edit's comment, or the changes highlights but it appeared in the new codes (not the old ones). I will remove this line, but I'm also lowkey writing this to remember this exists, so I can recover it if it turns out I'll need it. (A little of a mental note.)

Reply:
Nevermind, I see where it came from (you've moved it from the game.script into here). IDK why it doesn't say it was added in line 27.